R/carehomes_rt.R

```diff
@@ -28,7 +28,13 @@ carehomes_Rt <- function(step, S, p) {
 
   calculate_ev <- function(t, S, drop_carehomes) {
     ## Next-Generation-Matrix
-    ngm <- beta[t] * outer(mean_duration[, t], S[, t]) * p$m
+    m <- p$m
+    ages <- seq_len(p$N_age - 2)
```

Comment:
we also have this handy little helper for age bins
https://github.com/mrc-ide/sircovid2/blob/5c0e0004d02ac123003440bc0bcdf292cd13badc/R/carehomes.R#L338-L341

Reply:
That's for getting the age bins to work out the weightings for carehome workers, right?

Reply:
oh yeah! oops!